src/main/java/ch/heigvd/dai/commands/Expose.java

```diff
@@ -0,0 +1,64 @@
+package ch.heigvd.dai.commands;
+
+import ch.heigvd.dai.utilities.BmpFile;
+import ch.heigvd.dai.utilities.FileManipulator;
+import ch.heigvd.dai.utilities.MessageManipulator;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    name = "expose",
+    description = "Expose an hidden file/message from a BMP image.")
+public class Expose implements Callable<Integer> {
+  @CommandLine.ParentCommand private Root parent;
+
+  @Override
+  public Integer call() {
+    if (Files.exists(Paths.get(parent.getFilenameMessage()))) {
+      if (Files.isDirectory(Paths.get(parent.getFilenameMessage()))) {
```

Comment:
I'm not sure we're allowed to use these, since they come from `java.nio`? Open to your opinion